src/testing/code-debugging.md

```diff
@@ -677,12 +693,18 @@ used to reduce how much needs to be repainted.
 
 ### Focus tree
 
-To debug a focus or shortcut issue, you can dump the focus tree
-using [`debugDumpFocusTree()`][].
+To debug a focus or shortcut issue, dump the focus tree
+using the [`debugDumpFocusTree()`][] method.
 
-For example:
+#### Example 4: Focus tree dump
```

Comment:
Why does this have a header and where does the 4 come from? The previous dumps didn't have a header.